generate.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+execute_fastlane_snapshot() {
+    echo "Updating fastlane snapshot"
+    bundle exec fastlane snapshot update
+}
+
+check_bundle() {
+    echo "Checking for missing bundle dependencies"
+    bundle check
+    if [ $? -ne 0 ]
+    then
+        echo "Dependencies missing, running bundle install"
+        bundle install --path vendor/bundle && bundle update
+    else
+        echo "No dependencies missing, checking for update"
+        bundle update
+    fi
+}
+
+run_swiftformat() {
+    ./bin/swiftformat .
+}
+
+run_license_plist() {
+    ./bin/license-plist
```

Comment:
@lennet I am unsure as to how to use this license-plist, could you tell me the exact command? Right now, it generates empty plist file.

Reply:
Sorry I should have been more precise. We don't wanna run license-plist in swift format with the script. We are already doing that as part of our build phase. We want to update the binary that's stored in the repo. The updates can be find here https://github.com/mono0926/LicensePlist/releases and here https://github.com/nicklockwood/SwiftFormat/releases

Reply:
Oh ok, in that case, do you think `brew install/upgrade` would be better here? Since, as far as I know, to download from Releases from GitHub, we'd have to do authentication, which becomes tedious if you have two factor auth enabled.

Reply:
The latest release for a public repo doesn’t require authentication https://stackoverflow.com/a/54836319

Reply:
Yup, just found that out! It can be used without auth, but there is a rate limitation, which I don't think should be a problem in this case. 
Have updated the PR to download the releases @lennet, let me know if you have any ideas to make it better!

Reply:
Looks awesome! Just one small comment: Lets rename it to something like update_dependencies.sh to make it easier to understand without opening the script itself

Reply:
Sure, makes sense. Done!